core/ecschema-rpc/impl/package.json

```diff
@@ -37,7 +37,7 @@
     "@itwin/core-geometry": "workspace:*",
     "@itwin/core-backend": "workspace:*",
     "@itwin/core-common": "workspace:*",
-    "@bentley/itwin-client": "workspace:*"
+    "@bentley/itwin-client": "workspace:^3.0.0-dev.102"
```

Comment:
Did you need to make this change to get it working?

Reply:
No, not to fix the doc, but I was still seeing critical security issues due to immer and it was pointing to maplayers extension which I noticed did not have peer dependency explicitly set and many other uses of itwin-client did, so I change them all to match.  I was waiting for CI job to see if it actually got rid of the immer security issue.

Reply:
@calebmshafer  - I am still seeing `##[warning]CRITICAL Security Vulnerability: Prototype Pollution in immer in immer (from ..__..__extensions__map-layers>@bentley/react-scripts>react-dev-utils>immer)` so I will revert  the itwin-client peer dependency changes.